README.md

```diff
@@ -5,26 +5,25 @@ Back-tick template engine for JavaScript
 ## Usage
 
 - Install: `npm install tickplate`
-- Require: `const t = require('tickplate');`
-- Place tag `t` before templated string
+- Require: `const tpl = require('tickplate');`
+- Call `tpl` function like that: `tpl(filename, data, your_callback(err, data))`
```

Comment:
`your_callback` example is misleading. Better move the explanation of the function signature out of the inline code snippet, or, if you really want to leave it here, put it inside a `/* */`-style comment.